README.md

```diff
@@ -53,12 +50,12 @@ folder.
 | foreground | [port] [admin password] | 8080 random | Start tomcat in the foreground with the given port and password. |
 | background | [port] [admin password] | 8080 random | Start tomcat in the background with the given port and password. |
 
-### Notes
+## Notes
 
 Eventually we will support running on either Open JDK or Oracle JDK, but
```

Comment:
Based on the Tomcat Image and my [question there](https://github.com/Islandora-CLAW/claw-docker-tomcat/pull/2/files#r60860172), should this be edited as well?